### PR TITLE
style: edit model draft mode

### PR DESCRIPTION
**Pull request by asylves1**
# Description

Add Draft mode style

https://github.com/user-attachments/assets/746915ed-1bf3-452a-b84d-52e775ae3ac5

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue [hunk 1]
```diff
@@ -51,8 +51,10 @@
 						</Chip>
 					</aside>
 					<template v-if="!hideDropdown && outputOptions && selectedOutputId">
+						<section v-if="isDraft">There are unsaved changes</section>
 						<tera-output-dropdown
 							class="mx-2"
+							:class="isDraft ? 'draft' : ''"
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue:57]: The vue way: `:class="{'draft': isDraft}"`

File: packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue [hunk 2]
```diff
@@ -410,6 +426,7 @@ onMounted(async () => {
 		// Set default output which is the input (original model)
 		createOutput(originalModel);
 	}
+	hasCodeChange();
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue:429]: why is this checked for in the onMounted()?

Comment (asylves1) [reply packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue:429]: Not needed removed

File: packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue [hunk 3]
```diff
@@ -183,7 +177,9 @@ const defaultCodeText = '# This environment contains the variable "model" \n# wh
 const codeText = ref(defaultCodeText);
 const llmQuery = ref('');
 const llmThoughts = ref<any[]>([]);
-const isSaveForReuseDisabled = ref(false);
+
+const isDraft = ref(false);
+let saveModel = false;
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue:182]: I don't understand what this  variable saveModel is used for? can you provide some explanation?

**Review by mwdchang — APPROVED** [review]

**Review by blanchco — APPROVED** [review]
looks okay

**Pull request merged**